airflow/providers/hashicorp/CHANGELOG.rst

```diff
@@ -27,6 +27,19 @@
 Changelog
 ---------
 
+3.6.3
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``fix(hashicorp): Fix 'VAULT_TOKEN' support (#37337)``
```

Comment:
I marked this as fix rather than feature. It seems more like fixing something that should have work to begin with rather than adding new capability.

Reply:
+1